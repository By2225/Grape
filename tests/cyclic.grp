fun Int main() {
        Int b;

        b = 7;

        Int a = 5;

        Node<Int> start = '0';

        Graph<Int, String> g = << 
                start  -"h"-  '0'  -"i"-  '1',
                start  -"h"-  '0'  -"o"-  '1',
                start  -"s"-  '0'  -"u"-  start  -"p"-  '1'
        >>;

        prints("Hell oWorld ");

        List<Edge<String> > adjacent = g.outgoing(start);

        Int i = 0;
        while (i < size(adjacent)) {
                prints(adjacent[i].val);
                i = i + 1;
        }

        return 0;
}
