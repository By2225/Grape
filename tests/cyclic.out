Hell oWorld 
h
h
s
p
